style.css

```diff
@@ -55,23 +146,185 @@ ul {
   color: #172b4d;
 }
 
+.tertiary_color {
+  color: #3a4a42;
+}
+
+/* Buttons */
+
+.btn1 {
+  background-color: #36b37e;
+  height: 48px;
+  width: 122px;
+  border-radius: 4px;
+  padding: 12px;
+  border: none;
+  cursor: pointer;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  color: #fff;
+}
+
+.btn1:enabled {
+  background-color: #36b37e;
+}
+
+.btn1:hover {
+  background-color: #36b37f;
+}
+
+.btn1:active {
+  background-color: #008552;
+}
+
+.btn_resume {
+  background-color: #36b37e;
+  width: 159px;
+}
+
+/* Classes */
+
+.article_about_me p {
+  margin-bottom: 35px;
+  border-radius: 8px;
+  height: 474px;
+  width: 327px;
+}
+
+.article_card {
+  border-radius: 8px;
+  height: 352px;
+  width: 327px;
+}
+
+.article_content {
+  border-radius: 8px;
+  height: 222px;
+  padding: 16px;
+  width: 327px;
+}
+
+.article_works {
+  border-radius: 8px;
+  height: 474px;
+  width: 327px;
+}
+
+.border {
+  border: 1px solid #d6d4d4;
+  border-radius: 5px;
+}
```

Comment:
- [ ] As stated by the previous reviewer kindly give your cards the appropriate `background-color` 🙏 

Figma Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/206037934-6c5a4bdf-1f90-4119-81cb-6906b059cfab.png)  |  ![image](https://user-images.githubusercontent.com/74814780/206037900-adc72858-ce02-47cf-8442-d8e0a36b6143.png)